### PR TITLE
Added Acronis

### PR DESCRIPTION
**Pull request by Hime0698**

### CODE REVIEW

**Comment by Hime0698** [comment]
Huh.... think I might be having a tabs vs spaces type issue.... Ill see what I can do.

File: _data/sites.json [hunk 1]
```diff
@@ -200,6 +200,15 @@
...
     },
+    
+    {
+    	"name": "Acronis",
+    	"url": "https://account.acronis.com/#/profile/",
+    	"difficulty": "easy",
+    	"domains": [
+    		"acronis.com"
+    	]
+    },
```
Comment (tupaschoal) [inline _data/sites.json:211]:
```suggestion

    {
        "name": "Acronis",
        "url": "https://account.acronis.com/#/profile/",
        "difficulty": "easy",
        "domains": [
            "acronis.com"
        ]
    },    
```

**Comment by tupaschoal** [comment]
I think the suggestion I made might do it's

**Comment by Hime0698** [comment]
@tupaschoal Fixed most of it, except for one trailing spaces issue cause by copy and paste lol.

Should be fixed now, thanks for the assist.

**Review by tupaschoal — COMMENTED** [review]
While we're at it, any reason for not adding any notes? Is it clear how to delete the account as soon as you're on your profile page, like a big red button?

**Comment by Hime0698** [comment]
@tupaschoal  Yeah, small scroll down to the bottom and there is a bit button that says delete account.

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**